synapse/python_dependencies.py

```diff
@@ -221,3 +230,5 @@ def _check_requirement(dependency_string):
     import sys
 
     sys.stdout.writelines(req + "\n" for req in list_requirements())
+
+    check_requirements(dev=True)
```

Comment:
Does something run this module at some point? When is this used?

Reply:
No, currently nothing runs the module at this point, but this way, `python -m synapse.python_dependencies` can be suggested by someone from the synapse team to be ran by developers to check if their dependencies are alright, or if one of them is the wrong version.